src/main/java/domain/Cars.java

```diff
@@ -0,0 +1,61 @@
+package domain;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class Cars {
+    private final List<Car> cars = new ArrayList<>();
+    private static final String SPLIT_COMMA = ",";
+
+    public Cars(String carsName) {
+        splitCarsName(carsName);
+    }
+
+    public List<Car> getCars() {
+        return Collections.unmodifiableList(cars);
+    }
+
+    public void addCars(String carName) {
+        Car car = new Car(carName);
+        this.cars.add(car);
+    }
+
+    public void splitCarsName(String carsNames) {
+        String[] carName = carsNames.split(SPLIT_COMMA);
+        for (String name : carName) {
+            addCars(name);
+        }
+    }
```

Comment:
```suggestion
    public Cars(String carsName) {
        this(split(carsName));
    }

    private Cars(String[] carsName) {
        Arrays.stream(carsName)
            .map(Car::new)
            .forEach(car -> cars.add(car));
    }

    private static String[] split(String carsNames) {
        return carsNames.split(SPLIT_COMMA);
    }
```
1. splitCarsName 메서드가 Car를 생성하고 추가하는 기능까지 하고 있네요?
2. addCars와 같은 메서드를 추가하기보단 stream api 를 적극적으로 활용하는거는 어떨까요?
3. 객체 사용편의를 제공하다보면 다양한 인자를 받는 생성자가 도출되는 것은 자연스러운 현상입니다.